mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -758,37 +761,42 @@ func testAccMongoDBAtlasClusterAWSConfigdWithLabels(projectID, name, backupEnabl
 	`, projectID, name, backupEnabled, tier, region, labelsConf)
 }
 
-func testAccMongoDBAtlasClusterPrivateEndpointLinkConfig(projectID, providerName, region, vpcID, subnetID, securityGroupID string) string {
+func testAccMongoDBAtlasClusterPrivateEndpointLinkConfig(awsAccessKey, awsSecretKey, projectID, providerName, region, vpcID, subnetID, securityGroupID string) string {
 	return fmt.Sprintf(`
+		provider "aws" {
+			region     = "us-east-1"
+			access_key = "%[1]s"
+			secret_key = "%[2]s"
+		}
 		resource "mongodbatlas_private_endpoint" "test" {
```

Comment:
could you add space between resources to get a better format?

Reply:
sure, it's done